ipaserver/install/certs.py

```diff
@@ -581,8 +581,16 @@ def export_pkcs12(self, pkcs12_fname, pkcs12_pwd_fname, nickname=None):
         ])
 
     def create_from_cacert(self):
+        """
+        Ensure that a CA chain is in the NSS database.
+
+        If an NSS database already exists ensure that the CA chain
+        we want to load is in there and if not add it. If there is no
+        database then create an NSS database and load the CA chain.
+        """
         cacert_fname = paths.IPA_CA_CRT
-        if os.path.isfile(self.certdb_fname):
+
+        if self.nssdb.exists():
```

Comment:
Now we are missing the create, right?

Shouldn't it be something like:
```python
if not self.nssdb.exists():
  self.nssdb.create_db()
... # the rest
```

Reply:
It not self.nssdb.exists() it falls down to:

        self.create_passwd_file()
        self.create_certdbs()
        self.load_cacert(cacert_fname, IPA_CA_TRUST_FLAGS)